packages/widgets/src/lib/components/InferenceWidget/shared/WidgetWrapper/WidgetWrapper.svelte

```diff
@@ -35,6 +35,10 @@
 			if (modelLoadInfo?.state === "TooBig") {
 				updateWidgetState(model.id, "isDisabled", true);
 			}
+
+			if (!$tgiSupportedModels) {
+				tgiSupportedModels.set(await getTgiSupportedModels(apiUrl));
+			}
```

Comment:
Not a huge fan of this but it seems to work

I'm open to suggestions on how to grab the list of supported models from the Inference API though